rollup.config.js

```diff
@@ -1,12 +1,91 @@
-import nodeResolve from "rollup-plugin-node-resolve";
-import commonjs from "rollup-plugin-commonjs";
-import { terser } from "rollup-plugin-terser";
+const resolve = require("rollup-plugin-node-resolve");
+const commonjs = require("rollup-plugin-commonjs");
+const livescript = require("rollup-plugin-livescript");
+const pegjs = require("rollup-plugin-pegjs");
+const extensions = require("rollup-plugin-extensions");
+const { terser } = require("rollup-plugin-terser");
+const replace = require("rollup-plugin-re");
+const glob = require("glob");
+const path = require("path");
+const ignoreErrors = require('./src/plugin-pegjs.js');
 
-export default {
-    input: "dist/index.js",
-    output: {
-        format: "esm",
-        file: "dist/latex.esm.js"
-    },
-    plugins: [nodeResolve(), commonjs(), terser()]
-}
+const plugins = (format) => [
+    extensions({extensions: [".js", ".ls", ".pegjs"]}),
+    pegjs({plugins: [ignoreErrors], target: "commonjs", format: "commonjs"}),
+    livescript(),
+    replace({
+        patterns: [
+            {
+                match: /\/src\//,
+                test: /requireAll\("([^"]+)"\)/g,
+                replace(_, pattern) {
+                    return "({"
+                        + glob
+                            .sync(pattern)
+                            .map(f=>`"${path.basename(f)}": require("${path.resolve(f)}")`)
+                            .join(',')
+                        + "})";
+                }
+            }
```

Comment:
please write one line of documentation why this part is needed - it's not really obvious and a short explanation makes it easier to understand